### DISCUSSION TITLE
［求助］想抽简，但不知道怎么养

### DISCUSSION THREAD

**Discussion by pingwuping** [share121/inter-knot#2990] [其他]
如标题所说，我这位可怜的绳匠因为二队没有主c所以想要抽简，但抽出来养的不好还是没用，求各位绳网大佬给点建议。
Q：驱动盘如何选择，是四蓝调还是四物理？
       四号盘选暴击爆伤还是异常精通？
       简一定要绑专武吗？
       <img src="https://bbs-static.miyoushe.com/static/2023/11/16/dfc7ccf0203331b18508e05f4ccd7834_8488698876932024893.png" alt="本-疯狂计算" width="75" >
      其他关于配队、手法和武器替代之类的问题也欢迎各位提出建议和讨论，不过不要吵架哦<img src="https://bbs-static.miyoushe.com/static/2023/11/16/232b6fd4f34212bd5f8f854b821b51ca_8087743279879108307.png" alt="妮可-哈哈" width="75" >
      感谢各位提出建议的大佬<img src="https://bbs-static.miyoushe.com/static/2023/11/16/53069b115197426eff86537576a35d75_2306411376722926481.png" alt="外卖员邦布-出发" width="75" >
![1723995759703](https://github.com/user-attachments/assets/cde845bd-bea7-42ca-a13c-2d33e3735eee)

**Comment by miaochu1** [comment]
现在还没出，没啥攻略，可以等一手攻略<img src="https://bbs-static.miyoushe.com/static/2023/04/28/f2f1488ffbfff3fa96d4db6047e5f409_6776930953450600054.png" alt="米仔_不了" width="75" >

**Comment by chihung3927** [comment]
简 赛斯 露西
简主要是4自由  副词条主要是要异常精通跟攻击% 武器用双生或者唇彩也可以
